src/itest/java/uk/gov/companieshouse/insolvency/data/AbstractMongoConfig.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.companieshouse.insolvency.data;
+
+import org.springframework.test.context.DynamicPropertyRegistry;
+import org.springframework.test.context.DynamicPropertySource;
+import org.testcontainers.containers.MongoDBContainer;
+import org.testcontainers.utility.DockerImageName;
+
+public class AbstractMongoConfig {
+
+    public static final MongoDBContainer mongoDBContainer = new MongoDBContainer(
+            DockerImageName.parse("mongo:4.0.10"));
```

Comment:
question (non-blocking): Not sure about this, but In the docker compose file the image is `mongo:3.6`, do those need to match?

Reply:
I see it was copied over from `RepositoryITest` perhaps it is fine